include/exadg/incompressible_navier_stokes/user_interface/parameters.cpp

```diff
@@ -485,6 +489,25 @@ Parameters::check(dealii::ConditionalOStream const & pcout) const
                 dealii::ExcMessage("Not implemented."));
   }
 
+  // CONSISTENT SPLITTING SCHEME
+  if(temporal_discretization == TemporalDiscretization::BDFConsistentSplittingScheme)
+  {
+    AssertThrow(spatial_discretization != SpatialDiscretization::HDIV,
+                dealii::ExcMessage("Not implemented."));
+
+    AssertThrow(order_extrapolation_pressure_rhs <= order_time_integrator,
+      dealii::ExcMessage("Invalid parameter order_extrapolation_pressure_rhs!"));
+
+    AssertThrow(order_extrapolation_pressure_nbc <= order_time_integrator,
```

Comment:
Is this also used? This refers to the $J_p$ used in the paper, similar to the dual splitting scheme.
But we use it for the entire RHS of the PPE for the consistent splitting scheme.
Can you please explain this?

Reply:
@dominiktassilostill this one is still open :)
please just resolve if true :+1: